.github/workflows/update-configuration.yml

```diff
@@ -0,0 +1,58 @@
+name: "Update Configuration"
+
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  update:
+    name: "Update Configuration in manifest.json"
+    runs-on: ubuntu-latest
+    permissions: write-all
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20.10.0"
+
+      - name: Install deps and run configuration update
+        run: |
+          yarn install --immutable --immutable-cache --check-cache
+          yarn tsc --noCheck --project tsconfig.json
+
+      - name: Update manifest configuration using GitHub Script
+        uses: actions/github-script@v7
+        with:
+          script: |
```

Comment:
Probably best to compile the js with `ncc` 

This will allow us to skip on installing node and all the dependencies and allow for a near instant cold start time 

In exchange for possibly less maintainability because we won't have the original source code here

Reply:
I think this should be its own task because it would probably require changes in all the plugins as well. Also specifically this Action can be slow it doesn't really matter, it just generates the JSON.